script.js

```diff
@@ -1,6 +1,33 @@
+
+async function getBoredAPI(){
+    const textBox = document.querySelector('#user-input')
+
+    var number = textBox.value;
+    console.log(number)
+    const boredURL = `https://www.boredapi.com/api/activity?participants=${number}`;
+    const response = await fetch(boredURL);
+    boredData = await response.json();
+
+    unpackInfoFromBored(boredData);
+
+    
+}
+
+function unpackInfoFromBored(activity){
+    
+    console.log(`An activity you can do is ${activity.activity}`)
+
+   const activityBox =  document.querySelector(`#activity-text`);
+   activityBox.textContent = activity.activity
+
+}
+
```

Comment:
This project meets the requirements, but I believe you were capable of a lot more!